source/process/documentation-UItext-guidelines.rst

```diff
@@ -0,0 +1,196 @@
+User Interface Text Guidelines
+===============================
+This is the Mattermost style guide for text used in the Mattermost product. Use these guidelines to ensure the text on the Mattermost user interface (UI) is consistent and clear.  These guidelines cover two types of UI text:
```

Comment:
```suggestion
This is the Mattermost style guide for text used in the Mattermost product. Use these guidelines to ensure text on the Mattermost user interface (UI) is consistent and clear.  These guidelines cover two types of UI text:
```

Reply:
Thank you @jasonblais. I made this change.